### DISCUSSION TITLE
Problem of inaccurate subtitles for long videos

### DISCUSSION THREAD

**Discussion by S-CBuger** [openai/whisper#1811] [Q&A]
I used videos that were 8 hours long as input, and the resulting subtitle timestamps became more and more advanced over time, resulting in inaccurate subtitle timestamps.
Tried again with a 2 hour long video and still had the subtitle timestamps getting ahead of schedule.
It seems that you can get good results for short duration videos and have problems with the timestamps of the subtitles for long duration videos?
Can you tell me what I should do to fix this?

**Comment by glangford** [comment]
There is no guaranteed fix for the drifting of timestamps for long duration audio/video, but the best option available is to try 
`--word_timestamps True` for timestamp accuracy.

See also

* https://github.com/openai/whisper/discussions/1770
* https://github.com/openai/whisper/discussions/435
